homeassistant/components/reolink/host.py

```diff
@@ -84,6 +87,20 @@ def __init__(
         self._long_poll_task: asyncio.Task | None = None
         self._lost_subscription: bool = False
 
+    @callback
+    def async_register_update_cmd(self, cmd: str, channel: int | None = None) -> None:
+        """Register the command to update the state."""
+        self._update_cmd[cmd][channel] += 1
+
+    @callback
+    def async_unregister_update_cmd(self, cmd: str, channel: int | None = None) -> None:
+        """Unregister the command to update the state."""
+        self._update_cmd[cmd][channel] -= 1
+        if self._update_cmd[cmd][channel] <= 0:
```

Comment:
```suggestion
        if not self._update_cmd[cmd][channel]:
```

It should never go below 0 since it all happens in the event loop and there is no way for it to be decremented except here

Reply:
It would be a bug if its < 0

Reply:
I know, but I just wanted to be rather safe then sorry ;)